packages/design-system/.storybook/docs/tokensDocHelpers/components/DefinitionList/SizingScale/SizingScale.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Token } from '../../../../../../src/tokens/types';
+
+import styles from './SizingScale.module.scss';
+import { getScssName } from '../../../TokenFormatter';
+import classnames from 'classnames';
+import { SizedIcon, Tooltip } from '../../../../../../src';
+
+function SizeBlock({ token }: { token: Token }) {
+	const [isCopied, setIsCopied] = useState(false);
+
+	const handleCopy = useCallback(() => {
+		if (navigator.clipboard) {
+			navigator.clipboard
+				.writeText(getScssName(token?.name))
+				.then(() => {
+					setIsCopied(true);
+				})
+				.catch(err => {
+					// eslint-disable-next-line no-console
+					console.log('Something went wrong', err);
+				});
+		}
+	}, [getScssName(token?.name)]);
```

Comment:
There are multiple duplicate `handleCopy` and button code. It seems that there are multiple styles, perhaps we can put that together, perhaps not

Reply:
Yeah, not really sadly. But I could try and move the logic of it in a hook. It's just extra work for what's just doc :D